test.e2e/spec/services.spec.js

```diff
@@ -515,7 +515,19 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
             return getNew('Customer').getCurrent().$promise;
           })
           .then(function(user) {
+            var LoopBackAuth = $injector.get('LoopBackAuth');
             expect(user.email).to.equal('persisted@example.com');
+            expect(LoopBackAuth.rememberMe).to.equal(true);
+          })
+          .catch(util.throwHttpError);
+      });
+
+      it('`undefined` rememberMe resolves to `true`', function() {
```

Comment:
Nitpick: the test case name should read like an English sentence starting with `it`. A better test name would be `it('resolves undefined rememberMe to true')` .